server/lib/object-storage/shared/object-storage-helpers.ts

```diff
@@ -25,6 +25,13 @@ type BucketInfo = {
   PREFIX?: string
 }
 
+type CommandInput =  {
```

Comment:
You don't need to create a struct, you can use `PutObjectCommandInput` (import it from aws sdk)

Reply:
True enough. I didn't even look.